app/views/orgs/rosters/import_from_lms.html.erb

```diff
@@ -15,6 +15,7 @@
               <dt class="Box-subtitle">Select from the choices below</dt>
             </dl>
             <ul>
+            <%= hidden_field_tag "lms_user_ids", @identifiers.[:"User IDs"] %>
```

Comment:
Could you explain what's going on in `@identifiers.[:"User IDs"]`?

Reply:
Yeah, it looks weird. That is because of how we are setting `@identifiers`. 

https://github.com/education/classroom/blob/master/app/controllers/orgs/rosters_controller/lti_dependency.rb#L17-L21

Reply:
Changed the approach of this, so it is more readable